CHANGES.md

```diff
@@ -12,6 +12,7 @@
 * The download-speed label now shows a 3 second average.
 * `hasher` will no longer blindly overwrite an existing bundleinfo.json but instead error out.
 * `hasher` will now exit with an error when the `pathToHash` has no files to hash.
+* `timestamps.json` is ignored, if it is corrupt.
```

Comment:
Should be a fix and not a feature, or?

Because trivrost was unable to start *at all* if the file was once corrupted.